### DISCUSSION TITLE
How to run the create-hono command in Deno

### DISCUSSION THREAD

**Discussion by kay-ws** [honojs/hono#1553] [Q&A]
I don't know how to run the create-hono command in Deno.

https://hono.dev/getting-started/deno says the following
`npm create hono@latest my-app`

There is no npm command in Deno. What should I do?

**Comment by EdamAme-x** [comment]
So when you create it, Deno will appear as an option.

You can set it up from there.

**Comment by yusukebe** [comment]
You can do it with this command:

```txt
deno run -A npm:create-hono my-app
```
